src/customNodeLibraryInternalManagerWrapper.hpp

```diff
@@ -0,0 +1,40 @@
+//*****************************************************************************
+// Copyright 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#pragma once
+
+#include <iostream>
+#include <memory>
+
+namespace ovms {
+
+typedef int (*deinitialize_fn)(void*);
```

Comment:
Type re-definition.
This is already defined in another header.
This is bad, if we will ever change the `deinitialize_fn` prototype (add params or change types) we will need to do it in both places.